homework2_count_vow.py

```diff
@@ -0,0 +1,73 @@
+# Task description:
+# The English language has five vowels: A, E, I, O, and U
+# Please count each vowel occurence in text bellow
+# ( sum of lower and capital cases)
+# and write output as table smth like this
+# -----------------
+# | vowel | count |
+# -----------------
+# |   a   |  11   |
+# |   e   |  23   |
+#   ...
+# -----------------
+
+# then modify text where each vowel replaced with
+# A->À;  a->à ; E-> É ; e->é; I->Í , i->í ; O->Ó ; o->ó; U->Ú; u->ú
+# ex. "Í wàndéréd lónély...."   and print it
+
+
+poem_text = """I wandered lonely as a cloud
+That floats on high o'er vales and hills,
+When all at once I saw a crowd,
+A host, of golden daffodils;
+Beside the lake, beneath the trees,
+Fluttering and dancing in the breeze. 
+
+Continuous as the stars that shine
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance."""
+
+sep_num = 27
+print("-" * sep_num)
+print(f"| {'vowel':^10} | {'count':^10} |")
+print("-" * sep_num)
+count_a = poem_text.count('A') + poem_text.count('a')

```

Comment:
Here as trick you can reduce code twice just by poem.lowercase()  and then count_a = poem_text.count('a')